lib/util/cache/repository/http-cache.ts

```diff
@@ -0,0 +1,33 @@
+import is from '@sindresorhus/is';
+import { DateTime } from 'luxon';
+import { GlobalConfig } from '../../../config/global';
+import { logger } from '../../../logger';
+import { HttpCacheSchema } from '../../http/cache/schema';
+
+export function cleanupHttpCache(cacheData: unknown): void {
+  if (!is.plainObject(cacheData) || !is.plainObject(cacheData['httpCache'])) {
+    logger.trace('cleanupHttpCache: invalid cache data');
```

Comment:
```suggestion
    logger.warn('cleanupHttpCache: invalid cache data');
```

Isn't this a rare and unlikely event? (if you exclude empty/null cacheData)